certificate/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
For `localhost` certificates I thought this was safe. My reference was `browsersync`: https://github.com/BrowserSync/browser-sync/blob/master/lib/server/certs/server.crt